_data/people_list.yml

```diff
@@ -0,0 +1,618 @@
+  github: julien-schanz
+  status: active
+
+- name: Johannes Schmitt
+  affiliation: Ruhr-Universität Bochum
+  website: https://joschmitt.eu
+  email: johannes.schmitt@ruhr-uni-bochum.de
+  github: joschmitt
+  paid_by_dfg: false
+  status: active
+
+- name: Hans Schönemann
+  affiliation: University of Kaiserslautern-Landau
+  website: https://math.rptu.de/ags/agag/personen/mitglieder
+  email: hans.schoenemann@math.rptu.de
+  github: hannes14
+  status: active
+
+- name: Benjamin Schröter
+  affiliation: KTH Stochkholm
+  website: https://people.kth.se/~schrot
+  email: schrot@kth.se
+  github: bschroter
+  status: active
+
+- name: Justus Springer
+  affiliation: Universität Tübingen
+  website: https://www.math.uni-tuebingen.de/de/forschung/algebra/personen/justus-springer
+  email: justus.springer@uni-tuebingen.de
+  github: justus-springer
+  status: active
+
+- name: Jan Philipp Thiele
+  affiliation: WIAS Berlin
+  github: jpthiele
+  website: https://wias-berlin.de/~thiele/
+  email: janphilipp.thiele@wias-berlin.de
+  status: active
+
+- name: Andrew P. Turner
+  affiliation: University of Pennsylvania
+  website: https://apturner.net/
+  email: turnerap@sas.upenn.edu
+  github: apturner
+  status: active
+
+- name: Grégory Vanuxem
+  github: gvanuxem
+  status: active
+  comment: Who ?
+
+- name: Laura Voggesberger
+  affiliation: Ruhr-Universität Bochum
+  website: https://www.ruhr-uni-bochum.de/ffm/Lehrstuehle/Lehrstuhl-VI/voggesberger.html
+  email: laura.voggesberger@ruhr-uni-bochum.de
+  github: voggesbe
+  status: active
+
+- name: Marcel Wack
+  affiliation: TU Berlin
+  website: https://page.math.tu-berlin.de/~wack/
+  email: wack@math.tu-berlin.de
+  github: Sequenzer
+  status: active
+
+- name: Lena Weis
+  affiliation: TU Berlin
+  website: https://page.math.tu-berlin.de/~weis/
+  email: weis@math.tu-berlin.de
+  github: lenweis
+  status: active
+
+- name: Yichao Yu
+  affiliation: Duke University
+  email: yyc1992@gmail.com
+  github: yuyichao
+  status: active
+  comment: Who ?
+
+- name: Matthias Zach
+  affiliation: University of Kaiserslautern-Landau
+  website: https://math.rptu.de/ags/agag/personen/mitglieder
+  email: zach@rptu.de
+  github: HechtiDerLachs
+  status: active
+
+- name: Mohamed Barakat
+  affiliation: University of Siegen
+  website: https://mohamed-barakat.github.io/
+  email: mohamed.barakat@uni-siegen.de
+  github: mohamed-barakat
+  status: retired
+
+- name: Reimer Behrends
+  affiliation: University of Kaiserslautern-Landau
+  github: rbehrends
+  status: retired
+  comment: Parallelization, GAP garbage collector interface
+
+- name: Taylor Brysiewicz
+  affiliation: University of Western Ontario
+  website: https://sites.google.com/view/taylorbrysiewicz/home
+  email: tbrysiew@uwo.ca
+  github: tbrysiewicz
+  status: retired
+
+- name: Giovanni De Franceschi
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  comment: Group theory - who ?
```

Comment:
did a postdoc in KL around 2021 (plus minus, would have to look it up, but probably good enough).

BTW I think it's a great idea you started to add years to some of this info